src/io/flutter/inspector/WidgetPerfTipsPanel.java

```diff
@@ -155,10 +153,14 @@ private void updateTip() {
 
   ArrayListMultimap<TextEditor, PerfTip> tipsPerFile;
 
-  private void showPerfTips(ArrayList<PerfTip> tips) {
+  private void showPerfTips(@NotNull ArrayList<PerfTip> tips) {
     perfTips.removeAll();
-    final PerfTip lastMainTip = currentTips != null && currentTips.size() > 0 ? currentTips.get(0) : null;
-    currentTips = tips;
+
+    final PerfTip lastMainTip = !currentTips.isEmpty() ? currentTips.get(0) : null;
```

Comment:
is lastMainTip used?

Reply:
No, removed (apparently it hadn't been used before the refactor either).